src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx

```diff
@@ -4,22 +4,46 @@ import "react-native"
 import * as renderer from "react-test-renderer"
 import { BidResult } from "../BidResult"
 
-it("renders winning screen properly", () => {
-  jest.useFakeTimers()
+const saleArtwork = {
+  current_bid: {
+    amount: "CHF10,000",
+    cents: 1000000,
+    display: "CHF 10,000",
+  },
+  sale: {
+    live_start_at: "2022-01-01T00:03:00+00:00",
+    end_at: "2022-05-01T00:03:00+00:00",
+  },
+}
+describe("BidResult component", () => {
+  Date.now = jest.fn(() => 1525983752116)
+  it("renders winning screen properly", () => {
+    jest.useFakeTimers()
 
-  const bg = renderer.create(<BidResult winning />).toJSON()
+    const bg = renderer.create(<BidResult winning sale_artwork={saleArtwork} navigator={jest.fn() as any} />).toJSON()
 
-  expect(bg).toMatchSnapshot()
-  expect(setInterval).toHaveBeenCalledTimes(1)
-})
+    expect(bg).toMatchSnapshot()
+    expect(setInterval).toHaveBeenCalledTimes(1)
```

Comment:
I understand what this doing, but I don't think this is the approach we want to take. Our goal in this test is to test the `BidResult` component, but this expectation is really testing a child component. Can we mock the timer component so we can get reproducible snapshots?